var/spack/repos/builtin/packages/py-psyclone/package.py

```diff
@@ -8,23 +8,28 @@
 
 class PyPsyclone(PythonPackage):
     """
-    Code generation system developed to support domain-specific languages
-    (DSLs) for finite element, finite volume and finite difference codes.
-    Notably, it is used in the LFRic Project from the UK Met Office, and
-    it also supports the GOcean (2D, finite difference) DSL.
+    A source-to-source Fortran compiler designed to programmatically optimise,
+    parallelise and instrument HPC applications (written in Fortran) via user-
+    provided transformation scripts. Additionally, PSyclone supports the
+    development of kernel-based, Fortran-embedded DSLs and is used in the UK
+    Met Office's next-generation modelling system, LFRic.
 
     """
 
     # Links
     homepage = "https://github.com/stfc/PSyclone"
     git = "https://github.com/stfc/PSyclone.git"
-    pypi = "PSyclone/PSyclone-2.5.0.tar.gz"
+    pypi = "PSyclone/PSyclone-3.0.0.tar.gz"
+
+    # Maintainers
+    maintainers("arporter", "sergisiso", "TeranIvy", "hiker")
 
     # License
     license("BSD-3-Clause")
 
     # Releases
     version("develop", branch="master")
+    version("3.0.0", tag="v3.0.0")
```

Comment:
Shouldn't this pick up the sha from pypi rather than a tag?

Reply:
It could. I don't know whether it matters or not but I'll update it (now that I've worked out where the sha is).